radareorg/cutter
# Add UI for managing layouts

**Your checklist for this pull request**


**Detailed description**

* Create simple dialog for renaming and deleting layouts
* Use editable combo-box for saving layout - simplifies editing existing layouts and communicates with the user doing that is possible.

**Test plan (required)**

![layout_manager](https://user-images.githubusercontent.com/7101031/82730994-83863300-9d0c-11ea-9b75-cce5e2810fd9.png)
![Screenshot from 2020-05-23 14-19-59](https://user-images.githubusercontent.com/7101031/82730996-8719ba00-9d0c-11ea-8901-9bffc171aa1a.png)
![Screenshot from 2020-05-23 15-42-57](https://user-images.githubusercontent.com/7101031/82730998-88e37d80-9d0c-11ea-838d-eed03f17bf4e.png)


* Open layout manager when there are no saved layouts
*  save a layout
* rename layout, make sure layout list in main window gets updated
* delete layout, make sure layout manager properly clears the list
* try renaming layout to name used by other layout
* try renaming layout to reserved name



**Closing issues**

Closes #694
